requirements-python3/dev-requirements.txt

```diff
@@ -33,7 +33,7 @@ commonmark==0.9.0         # via recommonmark
 concurrent-log-handler==0.9.12
 contextlib2==0.5.5
 coverage==4.5.1
-cryptography==2.6.1
+cryptography==2.7
```

Comment:
I'm confused how this was upgraded. `make requirements` is supposed to be strictly making sure that the requirements fit in with current definition in `*.in` files. It should only change when running `make upgrade-requirements` or changing the in files. (it didn't show up in https://github.com/dimagi/commcare-hq/pull/24519/)